crypto/bn/bn_rsa_fips186_5.c

```diff
@@ -251,12 +259,29 @@ int ossl_bn_rsa_fips186_4_gen_prob_primes(BIGNUM *p, BIGNUM *Xpout,
     return ret;
 }
 
+static ossl_inline
```

Comment:
Do we really need the `ossl_inline`?  Any half baked compiler will figure this out if it makes sense based on the user's options.  E.g. if I'm building -Os instead of -O3, not inlining is the better option.

Reply:
either way its a hint and I dont see the problem here..

Reply:
I still think we're prematurely optimising by including it.